fitbenchmarking/core/tests/test_results_fallback.py

```diff
@@ -0,0 +1,660 @@
+"""
+Tests for renaming col tags if jacobian and hessian experience fallback
+"""
+
+import unittest
+
+from fitbenchmarking.core.results_output import (_get_all_result_tags,
+                                                 _find_non_full_columns,
+                                                 _handle_fallback_tags,
+                                                 preprocess_data)
+
+from fitbenchmarking.core.tests.test_results_output import load_mock_results
+
+
+class EdgeCaseTests(unittest.TestCase):
```

Comment:
I think if this fails it will just be called `EdgeCaseTests` in the log - it might be worth choosing a longer name to help highlight which part of the code is being tested for edge cases.